core/src/main/java/io/grpc/HandlerRegistry.java

```diff
@@ -42,6 +44,16 @@
 public abstract class HandlerRegistry {
 
   /**
+   * Returns the {@link ServerServiceDefinition}s provided by the registry.
+   *
+   * @throws UnsupportedOperationException if not supported by implementation
+   */
+  @ExperimentalApi("https://github.com/grpc/grpc-java/issues/2222")
+  public List<ServerServiceDefinition> getServices() {
```

Comment:
Carl asked if I could change it from a `Collection` to a `List` or `Set`, partially to make things easier to test. But being able to return map.values() might qualify as a compelling reason to stick with `Collection`. Happy to go with whichever.